### ISSUE TITLE
Getting Error 14 on Heroku

### ISSUE THREAD

**Issue by samira-karimi** [miguelgrinberg/microblog#313]
Hi and thank you for your amazing course.

I have been trying to deploy my app (in which I had to make some changes because I did not intend to make a blog, so I am not using the git versions) on Heroku. The app works on my local host. Also, `git push heroku master` runs with no error but I keep getting error 14 (H14 - No web dynos running) when trying to open my website. I have attached my requirements.txt and here is my Procfile content:
`web: flask db upgrade; gunicorn aicapp:app`
I am not using babel and translation, so I deleted that part from Procfile.  As my terminal, I am using "Git Bash" on windows.
Running `heroku ps:scale web=1` as suggested by the Heroku website also generates an error:
> Scaling dynos... !
 !    Couldn't find that process type (web).

I appreciate any help or suggestions. Thank you in advance.

[requirements.txt](https://github.com/miguelgrinberg/microblog/files/7909110/requirements.txt)
.

**Comment by miguelgrinberg** [comment]
With the little information you provide, my only guess is that your `Procfile` filename is misspelled in some way, so Heroku cannot find it.

**Comment by samira-karimi** [comment]
Thank you very much. It was incorrect at the beginning, then I had corrected the Procfile filename and run `git commit -am "message"`, and `git push heroku master` but it didn't work. Now, I deleted my application and started over and now it works.

I have another question if it's ok to ask in the same thread:
With changing sqlite to postgres the way you taught, will the data be available in the app automatically? Because I cannot log in with the credentials that I log in on the local host. So, I'm guessing the user data is not in my heroku app postgres database.
